### PR TITLE
cleanup(misc): disable cypress component tests e2e tests

### PR DESCRIPTION
**Pull request by leosvelperez**
Disabled until https://github.com/cypress-io/cypress/issues/25568 is addressed.

### CODE REVIEW

File: e2e/angular-extensions/src/cypress-component-tests.test.ts [hunk 1]
```diff
@@ -9,7 +9,9 @@ import {
   updateProjectConfig,
 } from '../../utils';
 import { names } from '@nrwl/devkit';
-describe('Angular Cypress Component Tests', () => {
+
+// TODO(leo): reenable once https://github.com/cypress-io/cypress/issues/25568 is addressed
+xdescribe('Angular Cypress Component Tests', () => {
```
Comment (barbados-clemens) [inline e2e/angular-extensions/src/cypress-component-tests.test.ts:14]: is it xdescribe or describe.skip?

File: e2e/react/src/cypress-component-tests.test.ts [hunk 2]
```diff
@@ -9,7 +9,8 @@ import {
   updateProjectConfig,
 } from '../../utils';
 
-describe('React Cypress Component Tests', () => {
+// TODO(leo): reenable once https://github.com/cypress-io/cypress/issues/25568 is addressed
+xdescribe('React Cypress Component Tests', () => {
```
Comment (barbados-clemens) [inline e2e/react/src/cypress-component-tests.test.ts:13]: is it xdescribe or describe.skip?

**Review by barbados-clemens — CHANGES_REQUESTED** [review]
noticed xdescribe but I believe it's supposed to be describe.skip
https://jestjs.io/docs/28.x/api#describeskipname-fn

**Review by barbados-clemens — APPROVED** [review]

File: e2e/angular-extensions/src/cypress-component-tests.test.ts (hunk 1 above)
Comment (leosvelperez) [reply e2e/angular-extensions/src/cypress-component-tests.test.ts:14]: both works

File: e2e/react/src/cypress-component-tests.test.ts (hunk 2 above)
Comment (leosvelperez) [reply e2e/react/src/cypress-component-tests.test.ts:13]: both works

**Pull request merged**